Sources/ProjectDescription/Project.swift

```diff
@@ -46,3 +50,71 @@ public struct Project: Codable, Equatable {
         dumpIfNeeded(self)
     }
 }
+
+// MARK: - Options
+
+extension Project {
+    /// Additional options related to the `Project`
+    public enum Options: Codable, Equatable {
+        /// Text settings to override user ones for currecnt project
+        case textSettings(TextSettings)
+    }
+}
+
+// MARK: - TextSettings
+
+extension Project.Options {
+    /// Text settings for Xcode project
+    public struct TextSettings: Codable, Equatable {
+        /// Use tabs over spaces
+        public let usesTabs: Bool?
+        /// Indent width
+        public let indentWidth: UInt?
+        /// Tab width
+        public let tabWidth: UInt?
+        /// Wrap lines
+        public let wrapsLines: Bool?
+        
+        public init(
+            usesTabs: Bool? = nil,
```

Comment:
I'd recommend to make this one internal and expose a static method:

```swift
.textOptions(...)

// vs

.init(....)
```
The former more semantic than a raw `init`.

Reply:
I'd also document all the attributes and methods in the `ProjectDescription` models. Those are surfaced by Xcode when the users are editing their projects.